react/src/components/card/components/SprkCardHighlightedHeader/SprkCardHighlightedHeader.js

```diff
@@ -1,64 +0,0 @@
-import React from 'react';
```

Comment:
Checked in spark-exports-react and we weren't exporting these internal components so there was no removal required from those files for these removed components.